tests/integration/components/bs-form-test.js

```diff
@@ -159,6 +161,29 @@ module('Integration | Component | bs-form', function(hooks) {
     assert.dom(`.${formFeedbackClass()}`).hasText('There is an error');
   });
 
+  test('Yields isSubmitting property', async function(assert) {
+    this.set('submit', () => {
+      return new RSVP.Promise((resolve) => {
+        later(() => {
+          resolve();
+        }, 10);
```

Comment:
Could also try awaiting next run loop here, which might be a little bit faster. But it's only 10ms and awaiting next run loop would fail as soon as `submit` function needs more than one runloop to execute.